integration-tests/test_motd.py

```diff
@@ -0,0 +1,52 @@
+import contextlib
+import os
+
+import pytest
+
+
+DOT_REGISTERED_PATH = "/etc/insights-client/.registered"
+DOT_UNREGISTERED_PATH = "/etc/insights-client/.unregistered"
+
+MOTD_PATH = "/etc/motd.d/insights-client"
+MOTD_SRC = "/etc/insights-client/insights-client.motd"
+
+
+pytestmark = pytest.mark.usefixtures("register_subman")
+
+
+@pytest.fixture
+def clean_system():
+    """Clean up the system before every test."""
+    with contextlib.suppress(OSError):
+        os.remove(DOT_REGISTERED_PATH)
+    with contextlib.suppress(OSError):
+        os.remove(DOT_UNREGISTERED_PATH)
+    with contextlib.suppress(OSError):
+        os.remove(MOTD_PATH)
+
+
+def test_motd(clean_system, insights_client):
```

Comment:
since `clean_system` is merely to do some pre-test work, maybe rather include it for the test without using it as parameter:
```python
@pytest.mark.usefixtures("clean_system")
def test_motd(insights_client):
```